src/main/java/com/fvv/bookstore/bean/Product.java

```diff
@@ -0,0 +1,122 @@
+package com.fvv.bookstore.bean;
+
+import java.util.Date;
+
+/**
+ * Bean abstract class for a Product object
+ * 
+ * @author Fatima Vasquez
+ * <p>Created on 2018</p>
+ * @version 1.0 	
+ *
+ */
+public abstract class Product {
+
+	private Long id;
+	private Double unitPrice;
+	private Integer stockQty;
+	private Date modificationDate;
+	
+	/**
+	 * Class constructor specifying the attributes
+	 * 
+	 * @param id of the Product
+	 * @param unitPrice of the Product
+	 * @param stockQty of the Product
+	 * @param modificationDate of the Product
+	 */
+	public Product(final Long id, final Double unitPrice, final Integer stockQty, 
```

Comment:
you re not supposed to have instances of isolated product. you can remove this constructor